modules/ossm-tutorial-bookinfo-adding-destination-rules.adoc

```diff
@@ -14,16 +14,15 @@ Before you can use the Bookinfo application, you have to add default destination
 ** If you did not enable mutual TLS:
 +
 
-[source,bash,subs="attributes"]
+[source,terminal]
 ----
 $ oc apply -n bookinfo -f https://raw.githubusercontent.com/Maistra/istio/maistra-{MaistraVersion}/samples/bookinfo/networking/destination-rule-all.yaml
 ----
 
 ** If you enabled mutual TLS:
 +
 
-[source,bash,subs="attributes"]
+[source,terminal]
```

Comment:
@JStickler Can you change this back to `[source,bash,subs="attributes"]`. It enables the version substitution in the command.